backend/app/tests/api/routes/collections/test_create_collections.py

```
@@ -41,15 +44,16 @@ def head_object(self, Bucket, Key):
+@patch("app.api.routes.collections.configure_openai")
+@patch("app.api.routes.collections.get_provider_credential")
 class TestCollectionRouteCreate:
     _n_documents = 5
 
-    @openai_responses.mock()
     def test_create_collection_success(
         self, client: TestClient, db: Session, normal_user_api_key_headers
     ):
-        store = DocumentStore(db)
+        # Setup test documents
+        store = DocumentStore(db, api_key_headers)
```

Ruff (0.12.2)

Undefined name `api_key_headers`

(F821)